fxtest/printer.go

```diff
@@ -20,7 +20,9 @@
 
 package fxtest
 
-import "go.uber.org/fx"
+import (
+	"go.uber.org/fx"
+)
```

Comment:
can revert this change.
```suggestion
import "go.uber.org/fx"
```